README.md

```diff
@@ -41,26 +41,25 @@ This data is collected, saved for individual time periods and transfered for fur
 
 - Access to an Industrial Edge Management System (IEM)
 - Onboarded Industial Edge Device on IEM
-- Installed System Configurators for Databus and S7 Connector
-- Installed System Apps Databus and S7 Connector
-- Installed Data Service
+- Installed System Configurators for Databus and OPC UA Connector
+- Installed System Apps Databus OPC UA Connector
+- Installed IIH Essentials
 - Installed Performance Insight
 - Edge device is connected to PLC
 - TIA portal project loaded on PLC (e.g. for filling application)
 - HTML5-capable Internet browser
 
 ### Used components
 
-- Industrial Edge Management (IEM) V1.1.16 (OS) V1.1.0-39
-  - IE Databus V1.1.15
-  - IE Databus Configurator V1.1.24
-  - S7 Connector V1.1.23
-  - S7 Connector Configurator V1.1.24
-  - Data Service V1.1.3
-  - Performance Insight V1.1.0
-- Industrial Edge Device V 1.1.0-32
-- TIA Portal V16
-- S7-PLCSIM Advanced V3.0 Upd 2
+- Industrial Edge Management (IEM) V1.14.10 (OS) V1.5.2-4
+  - Databus V2.2.0-3
+  - Databus Configurator V2.3.1-4
+  - OPC UA Connector V2.0.1-0
```

Comment:
Registry Service and Common Import Converter app is missing

Reply:
Additionally the [OPCUA_ADDTAGS.PNG](https://github.com/industrial-edge/performance-insight-getting-started/blob/Draft/docs/graphics/OPCUA_ADDTAGS.PNG) is not up tp date.

In the new version there is only one button (deploy), the stop button is not available anymore.
Furthermore it would be good if the whole name column would be visible, that the names can be read

Reply:
Hi norman,
I made the changes based on your comments.
Thanks

Reply:
Please also update the contribution license agreement section, as well delete the legacy contribution pdf